sql/core/src/test/scala/org/apache/spark/sql/execution/ui/UISeleniumSuite.scala

```diff
@@ -103,4 +113,16 @@ class UISeleniumSuite extends SparkFunSuite with WebBrowser {
       assert(planDetails.head.contains("TABLE_OR_VIEW_NOT_FOUND"))
     }
   }
+
+  test("Escape html is not necessary") {
+    spark = creatSparkSessionWithUI
+    val escape = (BASIC_ESCAPE.keySet().asScala.toSeq ++ ISO8859_1_ESCAPE.keySet().asScala ++
+      HTML40_EXTENDED_ESCAPE.keySet().asScala).mkString
+    val errorMsg = escapeJava(escape.mkString)
+    intercept[Exception](spark.sql(s"SELECT raise_error('$errorMsg')").collect())
```

Comment:
Although we don't care about the exceptions, shall we?
- `RuntimeException` instead of `Exception`?
- Check the error message?